app/views/helpers/lowAndHighBandInputs.scala.html

```diff
@@ -29,10 +29,14 @@
         suffix = Some(PrefixOrSuffix(
             content = Text("litres")
         )),
-        errorMessage = if (form("lowBand").hasErrors) {
-            Some(ErrorMessage(content = Text(messages(s"${form.errors.head.message}"))))
-        } else None,
-        value = form.data.get("lowBand")
+        errorMessage =
```

Comment:
why was this chnged for lowband, and also hint is not a message key

Reply:
This is the fix for https://jira.tools.tax.service.gov.uk/browse/DLS-8480
I'm not sure what you mean by 'also hint is not a message key'

Reply:
visuallyHiddenText = Some(messages("Error"))
should be 
visuallyHiddenText = Some(messages("messagekeyfrommessagefile"))

Reply:
does this need to be changed for highband aswell

Reply:
Fixed